h2o-algos/src/main/java/hex/glm/GLMModel.java

```diff
@@ -330,8 +337,14 @@ public void validate(GLM glm) {
             H2O.fail();
         }
       }
+      if (_missing_values_handling != null) {
+        if (!(_missing_values_handling instanceof DeepLearningModel.DeepLearningParameters.MissingValuesHandling) &&
+                !(_missing_values_handling instanceof MissingValuesHandling)) {
```

Comment:
the !(_missing_values_handling instanceof DeepLearningModel.DeepLearningParameters.MissingValuesHandling) part is for backward compatibility, right?

Reply:
exactly!